documentation/dev-portal/src/faq/integrations-faq.md

```diff
@@ -198,7 +198,7 @@ For the moment then yes, the mixnet is free to use. There are no limits on the a
 No, although we do recommend that apps that wish to integrate look into running some of their own infrastructure such as gateways in order to assure uptime. 
 
 ### How can I find out if an application is already supported by network requester services? 
-You can check the [default allowed list](https://nymtech.net/.wellknown/network-requester/standard-allowed-list.txt) file to see which application traffic is whitelisted by default. If the domain is present on that list, it means that existing [network requesters](https://nymtech.net/docs/nodes/network-requester.html) can be used to privacy-protect your application traffic. Simply use [NymConnect](../quickstart/nymconnect-gui.md) to connect to this service through the mixnet.
+You can check the [default allowed list](https://nymtech.net/.wellknown/network-requester/standard-allowed-list.txt) file to see which application traffic is whitelisted by default. If the domain is present on that list, it means that existing [network requesters](https://nymtech.net/docs/nodes/network-requester.html) can be used to privacy-protect your application traffic. 
 
 Currently we are undergoing changes on this policy under the name [Project Smoosh](https://nymtech.net/operators/faq/smoosh-faq.html) where a new type of node [Exit Gateway](https://nymtech.net/operators/legal/exit-gateway.html) will allow users to connect to much wider range of domains, restricted by our new [exit policy](https://nymtech.net/.wellknown/network-requester/exit-policy.txt). Follow the changes [here](https://nymtech.net/operators/faq/smoosh-faq.html#what-are-the-changes).
```

Comment:
Smoosh FAQ went to archive as well, here is what you may be looking for https://nymtech.net/operators/archive/faq/smoosh-faq.html#how-will-the-exit-policy-be-implemented

All the relevant info is implemented in the nym-node docs as a new standard.